instrumentation/grizzly-client-1.9/src/main/java/io/opentelemetry/auto/instrumentation/grizzly/client/GrizzlyClientResponseInstrumentation.java

```diff
@@ -60,7 +60,7 @@ protected boolean defaultEnabled() {
 
   @Override
   public String[] helperClassNames() {
-    return new String[] {packageName + ".ClientDecorator"};
+    return new String[] {packageName + ".GrizzlyClientTracer"};
```

Comment:
this should fix the muzzle failure 👍
```suggestion
    return new String[] {
      packageName + ".GrizzlyClientTracer", packageName + ".GrizzlyInjectAdapter"
    };
```

Reply:
Updated.  Thanks @trask.